app/Notifications/GalleryNftsChanged.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\AnonymousNotifiable;
+use Illuminate\Notifications\Messages\SlackAttachment;
+use Illuminate\Notifications\Messages\SlackMessage;
+use Illuminate\Notifications\Notification;
+
+final class GalleryNftsChanged extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    /**
+     * @param  array<int, array{'token_number': string, 'collection_id': int, 'name': string, 'updated_at': string}>  $deletedNfts
+     */
+    public function __construct(
+        public string $walletAddress,
+        public array $deletedNfts
```

Comment:
`$unassignedNfts` might be a better name here since we don't really delete them anymore.